### PR TITLE
Merge data transfer code.

### PR DESCRIPTION
**Pull request by ktlim**

### CODE REVIEW

File: apxfr/src/boto [hunk 1]
```diff
@@ -0,0 +1,400 @@
+# This file contains credentials and other configuration information needed
+# by the boto library, used by gsutil. You can edit this file (e.g., to add
+# credentials) but be careful not to mis-edit any of the variable names (like
+# "gs_access_key_id") or remove important markers (like the "[Credentials]" and
+# "[Boto]" section delimiters).
+#
+# This file was created by gsutil version 4.50 at 2020-05-15 05:17:17.
+#
```
Comment (hsinfang) [inline apxfr/src/boto:8]: Was this file generated by `gsutil config`? Would that command be more useful than this file?

File: apxfr/src/harness.py [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+    logging.info(f"Copying {source} to {temp / dest}")
+    subprocess.run(["cp", f"{source}", f"{temp / dest}"])
+    if compress:
+        logging.info(f"Compressing {temp / dest}")
+        subprocess.run(["fpack", f"{temp / dest}"])
+        dest = dest.with_suffix(".fits.fz")
+    return dest
+
+
+class Uploader(abc.ABC):
```
Comment (hsinfang) [inline apxfr/src/harness.py:137]: Some docstrings would be nice

**Review by hsinfang — APPROVED** [review]
If this will be used beyond the POC some more documentations would be helpful. But fine for the POC.

File: apxfr/src/boto (hunk 1 above)
Comment (ktlim) [reply apxfr/src/boto:8]: Then I'd have to install gsutil into the container, which I was trying to avoid.

**Pull request merged**